utility/proto/message.proto

```diff
@@ -4,156 +4,51 @@ package utility;
 option go_package = "github.com/pokt-network/pocket/utility/types";
 
 import "vote.proto";
-import "session.proto";
 import "google/protobuf/any.proto";
-import "google/protobuf/timestamp.proto";
+
+enum ActorType {
```

Comment:
Can we also make `StakingStatus` an enum? It's something I get confused about quite often throughout...

Reply:
Not worth it, it will be deprecated soon